core/src/main/java/org/apache/calcite/sql/SqlOperator.java

```diff
@@ -234,21 +234,21 @@ public int getRightPrec() {
   public abstract SqlSyntax getSyntax();
 
   /**
-   * Creates a call to this operand with an array of operands.
-   *
-   * <p>The position of the resulting call is the union of the <code>
-   * pos</code> and the positions of all of the operands.
+   * Creates a call to this operand with an array of operands and specifying
+   * whether it ignores nulls.
    *
    * @param functionQualifier function qualifier (e.g. "DISTINCT"), may be
+   * @param ignoreNulls       Whether ignore nulls
    * @param pos               parser position of the identifier of the call
    * @param operands          array of operands
    */
   public SqlCall createCall(
       SqlLiteral functionQualifier,
+      SqlLiteral ignoreNulls,
```

Comment:
This is an API-breaking change. Should add a new createCall function with `ignoreNulls` and deprecate the existing `createCall`.

Reply:
Got it.